pkg/controller/servicemesh/webhooks/mutation/controlplane.go

```diff
@@ -93,6 +93,25 @@ func (v *ControlPlaneMutator) Handle(ctx context.Context, req admission.Request)
 		}
 	}
 
+	// removing tracing by default on creation of v2.6 SMCP
+	if req.AdmissionRequest.Operation == admissionv1beta1.Create {
```

Comment:
We already have an "if" statement for "create" events, so please move the body of this condition to the previous one.